app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
```

Comment:
In my opinion, it is better to use a name for the variable that is different from the argument that the function accepts

Reply:
I agree, did not pay attention